quupe-RN/js/screens/SignUp/SignUp.js

```diff
@@ -48,10 +60,51 @@ const SignUp = props => (
                         />
                     )}
                 />
+                <Field
+                    name="terms"
+                    render={() => (
+                        <View style={styles.terms}>
+                            <RoundCheckbox
+                                onValueChange={() =>
+                                    props.dispatch(acceptTerms())
+                                }
+                                checked={props.acceptedTerms}
+                                style={styles.checkbox}
+                                size={20}
+                                backgroundColor={assetColors.darkGrey}
+                                borderColor={assetColors.darkGrey}
+                            />
+                            <Text style={styles.termText}>
+                                I agree to Quupe's{' '}
+                                <Text
```

Comment:
Text component should not have another Text component as a child. Please make appropriate changes.